Packs/CortexXDR/Integrations/XDR_iocs/XDR_iocs.py

```diff
@@ -220,6 +221,21 @@ def demisto_types_to_xdr(_type: str) -> str:
         return xdr_type
 
 
+def _parse_demisto_comments(custom_fields: dict, comment_field_name: str) -> str | None:
```

Comment:
```suggestion
def _parse_demisto_comments(custom_fields: dict, comment_field_name: str) -> Optional[str]:
```
Don't forget to add `from typing import Optional`

Reply:
isn't it implicitly imported on CommonServerPython? 🤔

Reply:
You are right